plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitView.java

```diff
@@ -49,46 +49,43 @@
      */
     void setMessage(@NotNull String message);
 
-    /** @return <code>true</code> if need to include all changes except from new files, and <code>false</code> otherwise */
-    boolean isAllFilesInclued();
-
     /**
-     * Set status of include changes to commit.
-     *
-     * @param isAllFiles
-     *         <code>true</code> need to include all changes except from new, <code>false</code> include all changes
+     * Returns <code>true</code> if need to add all changes to index except from new files before commit, and <code>false</code> otherwise
      */
-    void setAllFilesInclude(boolean isAllFiles);
+    boolean isAddAllExceptNew();
 
     /**
-     * Returns true if the project explorer selection must be included in the commited files.
-     * 
-     * @return true iff the selection mus tbe commited
+     * Set status of flag that represents add all changes to index except from new files before commit.
+     *
+     * @param addAllExceptNew
+     *         <code>true</code> if need to add all changes to index except from new files before commit,
+     *         <code>false</code> otherwise
      */
-    boolean isIncludeSelection();
+    void setAddAllExceptNew(boolean addAllExceptNew);
 
-    /**
-     * Sets the display of the include selection flag.
-     * 
-     * @param includeSelection is selection included
-     */
-    void setIncludeSelection(boolean includeSelection);
+    /** Returns true if the project explorer selection must be added to index before commit, and <code>false</code> otherwise. */
```

Comment:
Simple selection, it might not be from project explorer. Paraphrase this sentence.